README.md

```diff
@@ -1 +1,4 @@
+Contributions:
```

Comment:
@SudakovEugene you didn't fix the typo.
Should be `Contributors`, not `Contributions`.